eras/shelley-ma/test-suite/test/Test/Cardano/Ledger/Mary/Examples/Cast.hs

```diff
@@ -22,73 +22,73 @@ module Test.Cardano.Ledger.Mary.Examples.Cast
 where
 
 import Cardano.Ledger.Address (Addr (..))
+import Cardano.Ledger.Crypto (StandardCrypto)
 import Cardano.Ledger.Keys
   ( KeyPair (..),
     KeyRole (..),
   )
-import Test.Cardano.Ledger.EraBuffet (TestCrypto)
 import Test.Cardano.Ledger.Shelley.Utils (RawSeed (..), mkAddr, mkKeyPair)
 
 -- | Alice's payment key pair
-alicePay :: KeyPair 'Payment TestCrypto
+alicePay :: KeyPair 'Payment StandardCrypto
```

Comment:
Is this  `StandardCrypto` to `TestCrypto` related to split, or just an general improvement?

Reply:
This is general improvement. Whenever possible we should use `StandardCrypto`. THere are very few places were `TestCrypto` actually makes sense, in particular it is relevant when dealing with actual Blocks